### ISSUE TITLE
Not working on latest clone

### ISSUE THREAD

**Issue by sorenbug** [riking/joycon#35]
Followed instructions in readme and joycon does not connect as an input device or controller (tested via http://html5gamepad.com/)

**Comment by sorenbug** [comment]
Yes

**Comment by s5bug** [comment]
Switching to the `develop` branch also does not seem to work. I run `sync`, hold the sync button on my JoyCon, and nothing happens.

**Comment by qixils** [comment]
Is `jcdriver` running? Are you actually connecting to them via your system's Bluetooth settings? And are you pressing L + R on your JoyCons after `jcdriver` confirms your JoyCons have connected?

**Comment by s5bug** [comment]
jcdriver tells me to scan using it and not connect externally. When I do
connect externally, it is not registered in jcdriver. Pressing L and R do
nothing at any point.

**Comment by qixils** [comment]
jcdriver doesn't (as far as i'm aware) automatically connect your joycons to your computer. You need to manually enter your system's bluetooth settings, look for the joycons there while holding the sync button, and connect them to your system. You'll know jcdriver has detected your joycons if they start flashing a single light, instead of having the lights go up and down.

**Comment by riking** [comment]
Yeah, the auto connection is the feature I was working on, it was broken, so the changes I made were to disable it on `master`.

**Comment by riking** [comment]
Also confirm that you're running on Linux? I haven't written an OS X kernel driver yet, so it can't add a device on Mac.

**Comment by s5bug** [comment]
I am running on Arch Linux under i3wm and managing devices through
blueman-manager.

**Comment by s5bug** [comment]
I hold the sync button, pair through blueman-manager, and nothing happens in the console or with my joycon's player lights (even when pressing L+R). The joycon disconnects after a time.

**Comment by s5bug** [comment]
So, connecting it via the HID service connects it to the driver. I'll try connecting a second joycon.

Edit: Attempting to connect a second joycon via bluetooth results in `dmesg` spewing out
![image](https://user-images.githubusercontent.com/7751296/42738770-4073870a-883e-11e8-835c-32b655f2bebd.png)

I am able to connect a single joycon at a time without issue however trying to connect a second errors. Both joycons work properly via the driver (as a single joycon).

Blueman spits out 
![image](https://user-images.githubusercontent.com/7751296/42738833-6546d87e-883f-11e8-8246-e3c3368e8831.png)

**Comment by riking** [comment]
@sorenbug that's probably a bug with your bluetooth card / driver then :( You'll probably need to buy a different BT radio.

**Comment by s5bug** [comment]
Which is weird, because using mfosse's driver for windows works fine. I think it's a bug with linux bluez that nobody has bothered to resolve/look into yet.
